gluten-core/src/main/scala/io/glutenproject/expression/UnaryExpressionTransformer.scala

```diff
@@ -65,6 +66,30 @@ class NormalizeNaNAndZeroTransformer(
   }
 }
 
+class ExplodeTransformer(child: ExpressionTransformer, original: Explode)
```

Comment:
not need to specified new a transformer for `Explode`, right ?

Reply:
Yes, we need. It's logic is different from common implementations.
- Only support array type as input. 
- Output datetype must be nested type from child expression.